packages/legend-application-query/src/stores/QueryBuilderValueSpecificationBuilderHelper.ts

```diff
@@ -138,6 +142,7 @@ const isDefaultDatePropagationSupported = (
 export const buildPropertyExpressionChain = (
   propertyExpression: AbstractPropertyExpression,
   queryBuilderState: QueryBuilderState,
+  isDatePropagationNotSupported?: boolean,
```

Comment:
--> `disableDatePropagation`, let's also add documentation this flag. Every option-orientated param needs an explanation, else we could keep adding options like this on and on and at some point, we won't know what is each for anymore 😓